charts/cmc-claim-store/values.preview.template.yaml

```diff
@@ -66,7 +74,65 @@ cmc-citizen-frontend:
     java:
       releaseNameOverride: ${SERVICE_NAME}-ftr-tgl-api
       environment:
-        FEATURES_DB_HOST: ${SERVICE_NAME}-claim-store-postgres
+        FEATURES_DB_HOST: ${SERVICE_NAME}-ccd-postgres
   idam-pr:
     service:
       redirect_uri: https://www-${SERVICE_FQDN}/receiver
+
+ccd:
+  # TODO: update CCD chart with SIDAM
+  idamApiUrl: https://idam-api.aat.platform.hmcts.net
+  idamWebUrl: https://idam-web-public.aat.platform.hmcts.net
+  ingressHost: ${SERVICE_FQDN}
+  ingressIP: ${INGRESS_IP}
+  consulIP: ${CONSUL_LB_IP}
+  apiGateway:
+    s2sKey: ${API_GATEWAY_S2S_KEY}
+    idamClientSecret: ${API_GATEWAY_IDAM_SECRET}
+  userProfileApi:
+    authorisedServices: ccd_admin,ccd_data,ccd_definition,cmc_claim_store
+  dataStoreApi:
+    s2sKey: ${DATA_STORE_S2S_KEY}
+    s2sAuthorisedServices: cmc_claim_store,ccd_gw
+  definitionStoreApi:
+    s2sKey: ${DEFINITION_STORE_S2S_KEY}
+    s2sAuthorisedServices: ccd_admin,ccd_data,cmc_claim_store,ccd_gw
+  caseManagementWeb:
+    enabled: true
+    environment:
+      NODE_TLS_REJECT_UNAUTHORIZED: 0
+  adminWeb:
+    enabled: true
+    s2sKey: ${ADMIN_S2S_KEY}
+    idamClientSecret: ${ADMIN_WEB_IDAM_SECRET}
+    environment:
+      NODE_TLS_REJECT_UNAUTHORIZED: 0
+      # TODO: add these to chart-ccd
+      ADMINWEB_USER_ROLE_URL: http://${SERVICE_NAME}-definition-store-api/api/user-role
+      ADMINWEB_DELETE_DEFINITION_URL: http://${SERVICE_NAME}-definition-store-api/api/draft
+      ADMINWEB_ROLES_WHITELIST: 'ccd-import,ccd-import-validate'
+  importer:
+    userprofile:
+      enabled: true
+      jurisdictions:
+        - CMC
+      users:
+        - civilmoneyclaims+ccd@gmail.com|CMC|MoneyClaimCase|open
+      userProfileDatabaseHost: ${SERVICE_NAME}-ccd-postgres
+      userProfileDatabasePort: 5432
+      userProfileDatabaseUser: hmcts
+      userProfileDatabasePassword: hmcts
+      userProfileDatabaseName: user-profile
+    definition:
+      enabled: true
+      image: hmcts.azurecr.io/hmcts/cmc-ccd-definition-importer
+      definitionFilename: cmc-ccd.xlsx
+      definitions:  # TODO: change chart-ccd to allow either definitions or definitionFilename
```

Comment:
Is this going to be in a separate PR?

Reply:
Will be a change to ccd-docker-definition-importer - ill add you to PR

Reply:
already fixed in chart-ccd